bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
Yeah, this wouldn't work where there's only `libstdc++`. I thought about using `clang++` but wasn't sure I want to enable the whole bundle of implicit flags turned on by that, vs the actual need of linking to a C++. I can use [`have_library()`](http://www.rubydoc.info/stdlib/mkmf/MakeMakefile#have_library-instance_method) to check for `libc++` first then try `libstdc++` second.